lib/common.sh

```diff
@@ -426,3 +426,15 @@ function init_minikube() {
           --type network --config
     fi
 }
+
+#
+# Kill and remove the running ironic containers
+#
+function remove_ironic_containers() {
+  for name in ipa-downloader ironic ironic-api ironic-conductor ironic-inspector ironic-endpoint-keepalived dnsmasq httpd mariadb vbmc sushy-tools httpd-infra; do
+    sudo "${CONTAINER_RUNTIME}" ps | grep -w "$name$" || true
+    sudo "${CONTAINER_RUNTIME}" kill $name || true
+    sudo "${CONTAINER_RUNTIME}" ps --all | grep -w "$name$" || true
+    sudo "${CONTAINER_RUNTIME}" rm $name -f || true
```

Comment:
```suggestion
    sudo "${CONTAINER_RUNTIME}" ps | grep -w "$name$" && sudo "${CONTAINER_RUNTIME}" kill $name || true
    sudo "${CONTAINER_RUNTIME}" ps --all | grep -w "$name$" && sudo "${CONTAINER_RUNTIME}" rm $name -f || true
```

Reply:
I have tried this, shell will fail will following:
`In ./lib/common.sh line 435:
    sudo "${CONTAINER_RUNTIME}" ps | grep -w "$name$" && sudo "${CONTAINER_RUNTIME}" kill $name || true
                                                      ^-- SC2015: Note that A && B || C is not if-then-else. C may run when A is true.
In ./lib/common.sh line 436:
    sudo "${CONTAINER_RUNTIME}" ps --all | grep -w "$name$" && sudo "${CONTAINER_RUNTIME}" rm $name -f || true
                                                            ^-- SC2015: Note that A && B || C is not if-then-else. C may run when A is true.`

Reply:
I wanted to escape && and || to come together. Putting them separately works, but it will not fail silently:
`+ sudo podman ps
+ grep -w 'ipa-downloader$'
+ true
+ sudo podman kill ipa-downloader
Error: no container with name or ID ipa-downloader found: no such container
+ true`

Reply:
That is a false positive from the shellcheck. We really want C to run even if A succeeds. We want C to run whenever B fails. So disable shellcheck there